### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Application_ScreenReader: NVDA announces check box(Use credentials, Logout on exit, Use device code flow sign-in(preview)) names twice.

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#1924]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #NVDA; #Win32; #A11ySev3; #NonBenchmark;

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
Windows10

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to Setting menu on left pane and click on it.
3. Navigate to Application on right pane and  click on it.
4. "Application" screen gets opened.
5. Verify that all the elements of "Application" Screen are accessible and MAS Compliant.
6. Start NVDA.
7. Navigate using Tab key to check boxes(Use credentials, Logout on exit, Use device code flow sign-in(preview), verify NVDA announces check box name twice.

**​Actual:​**
While navigating using Tab key, NVDA announces check box(Use credentials, Logout on exit, Use device code flow sign-in(preview)) names twice.

**Same issue is observed in below Scenarios:**
1. **Data Explorers:**
* Here in this Scenario NVDA announces Check box name twice.
2. **Transfers:**
* Here in this Scenario NVDA announces Check box name twice.

**Expected:**
While navigating using Tab key, NVDA should announce check box(Use credentials, Logout on exit, Use device code flow sign-in(preview)) name only once.

**UserImpact:**
Screen reader users gets difficulty to understand the exact name of the check box if screen reader announces name of the check box twice.

**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[A11y_AzureToolsStorageExplorer_Application_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3657190/A11y_AzureToolsStorageExplorer_Application_ScreenReader.pptx)

**Comment by JasonYeMSFT** [comment]
This seems to be an NVDA bug. I have opened an issue against it.
https://github.com/nvaccess/nvda/issues/10512

**Comment by Suraj1200499** [comment]
@JasonYeMSFT Since the primary bug#10512 is in open State. We will take action on the bug#1924 Once the Primary bug is Closed.

**Comment by MRayermannMSFT** [comment]
Make sure to add to VPAT

**Comment by Alexandria-R** [comment]
This can be marked as Sev3, but not approved to close.

**Comment by jinglouMSFT** [comment]
In build 20200318.10, NVDA reads the title of the checkbox twice. For example, "Proxy, Proxy checkbox, unchecked".  However, the description of the checkbox is only read once.  On the same build, Narrator reads the title of the checkbox only once. For example, "Proxy checkbox, unchecked". However, Narrator doesn't read the description of the checkbox, such as for the proxy checkbox, it doesn't read "Enable proxy. Storage Explorer must be restarted for this setting to be applied".  So neither is good, but NVDA is a little better. 

Reading twice is an external bug in NVDA.

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
